### ISSUE TITLE
[Enhancement] List and explain any and all external dependencies

### ISSUE THREAD

**Issue by windowsrefund** [kyverno/website#585]
### Description

In light of [4226](https://github.com/kyverno/kyverno/issues/4226), I'd like to see a page focused on these questions:

- What external dependencies exist?
- For each of the above:
  - How does Kyverno use it?
  - Can it be bypassed? If so, how?
  - What is the impact of it not being available?

### Slack discussion

https://kubernetes.slack.com/archives/CLGR9BJU9/p1658238495198669

**Comment by chipzoller** [comment]
Agreed on all accounts. In this specific instance, neither we nor, as it would seem, many of the creators of that code were aware it behaved in such a manner. That said, we need to document these things in addition to providing a switch so that Kyverno is fully-functional especially in air-gapped environments.
